clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -46,9 +46,17 @@ class KubernetesContainerPort {
 
 @AutoClone
 @Canonical
+class KubernetesImageDescription {
+  String repository
+  String tag
+  String registry
+}
+
+@AutoClone
+@Canonical
 class KubernetesContainerDescription {
   String name
-  String image
+  KubernetesImageDescription imageDescription
```

Comment:
Seems like this would still read ok as `image`, but don't feel too strongly about it.